### PR TITLE
OTWO 4907 - Add ActiveAdmin for LicensePermission model

### PR DESCRIPTION
**Pull request by drubio1989**

### CODE REVIEW

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

File: app/models/license.rb [hunk 1]
```diff
@@ -61,4 +75,7 @@ def redo_project_licenses(project_licenses)
       pl.edits.undone_by_hamster.each { |edit| edit.redo!(Account.hamster) }
     end
   end
+
+  def add_required_license_permission(licence_permission)
+  end
```
Comment (PDegenPortnoy) [inline app/models/license.rb]: This looks like a thought in process that was never completed and can be removed.

File: app/admin/license_permission.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ActiveAdmin.register LicensePermission do
+  permit_params :name, :description, :icon
+end
```
Comment (PDegenPortnoy) [inline app/admin/license_permission.rb]: Was this all that was needed to add CRUD operations for the LicensePermission model? :-)

**Comment by drubio1989** [comment]
retest this please

**Comment by drubio1989** [comment]
retest this please

**Pull request merged**